controller/controller.go

```diff
@@ -23,12 +23,16 @@ const (
 	ingressAllowAnnotation   = "sky.uk/allow"
 	frontendSchemeAnnotation = "sky.uk/frontend-scheme"
 
-	stripPathAnnotation = "sky.uk/strip-path"
-	exactPathAnnotation = "sky.uk/exact-path"
+	// Deprecated: retained to maintain backwards compatibility.
+	legacyFrontendElbSchemeAnnotation = "sky.uk/frontend-elb-scheme"
+	stripPathAnnotation               = "sky.uk/strip-path"
+	exactPathAnnotation               = "sky.uk/exact-path"
 
-	backendTimeoutSeconds       = "sky.uk/backend-timeout-seconds"
-	proxyBufferSizeAnnotation   = "sky.uk/proxy-buffer-size-in-kb"
-	proxyBufferBlocksAnnotation = "sky.uk/proxy-buffer-blocks"
+	// Deprecated: retained to maintain backwards compatibility.
```

Comment:
I tried this, but you need a `// Deprecated: xxx` line for each `const` if you want IDEs to show them as deprecated, so it's best to leave each one with its non-deprecated version instead of having them together.